AirCasting/Extensions/Date+Miliseconds.swift

```diff
@@ -42,5 +42,25 @@ extension Date {
         let date = self
         return Date(timeIntervalSinceReferenceDate: (date.timeIntervalSinceReferenceDate / 3600.0).rounded(.towardZero) * 3600.0)
     }
+    
+    static func yearAgo() -> Date {
+        DateBuilder.getFakeUTCDate() - (365 * 24 * 60 * 60)
+    }
+    
+    static func beginingOfCurrentDayInSeconds() -> Double {
+        Calendar.current.startOfDay(for: DateBuilder.getFakeUTCDate()).timeIntervalSince1970
+    }
+    
+    static func beginingOfDayInSeconds(using date: Date) -> Double {
+        var calendar = Calendar.current
+        calendar.timeZone = TimeZone.utc
+        return calendar.startOfDay(for: date).timeIntervalSince1970
+    }
+    
+    static func endOfDayInSeconds(using date: Date) -> Double {
+        var calendar = Calendar.current
+        calendar.timeZone = TimeZone.utc
+        return calendar.startOfDay(for: date).timeIntervalSince1970 + (23 * 60 * 60 + 3540 + 59)
+    }
```

Comment:
`Date+Miliseconds.swift` is not a good place for "end of year" stuff IMO. Let's either create a new file for those or rename this one.

Also, those methods that take `Date` parameter should not do that, instead they should be non-static and operate on `self`.

Reply:
I have tried to improve those but not sure if I did it well, please comment if not 😅